target/linux/mediatek/dts/mt7986a-asus-rt-ax59u.dts

```diff
@@ -88,6 +88,8 @@
 	gmac0: mac@0 {
 		/* LAN */
 		compatible = "mediatek,eth-mac";
+		nvmem-cells = <&macaddr_factory_4>;
+		nvmem-cell-names = "mac-address";
```

Comment:
Those should not be above `reg`, see https://www.kernel.org/doc/html/next/devicetree/bindings/dts-coding-style.html#order-of-properties-in-device-node

Reply:
Ack, will order them correctly.